### PR TITLE
Chehov

### PR DESCRIPTION
**Pull request by Vira-tester**
package Writers;

public class Chehov {
    public static void main(String[] args) {
        System.out.println("Chehov");
    }
}

### CODE REVIEW

**Review by Vira-tester — COMMENTED** [review]
for Branch Writers

File: src/main/java/Writers/Chehov.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package Writers;
```
Comment (AidarMir) [inline src/main/java/Writers/Chehov.java:1]: Пушкин

**Review by AidarMir — COMMENTED** [review]
Привет

File: src/main/java/Writers/Chehov.java (hunk 1 above)
Comment (Vira-tester) [reply src/main/java/Writers/Chehov.java:1]: Эпрувни)))

**Pull request merged**